src/examples/flight/createorders/FlightCreateOrders.java

```diff
@@ -56,26 +60,28 @@ public static void main(String[] args) throws ResponseException {
     travelerArray[0] = traveler;
     System.out.println(travelerArray[0]);
 
-    Amadeus amadeus = Amadeus
```

Comment:
```> Task :checkstyleMain 
[ant:checkstyle] [ERROR] /Users/atsolakou/Desktop/workspace/amadeus-java/src/main/java/flightsearch/FlightSearch.java:30: Distance between variable 'amadeus' declaration and its first usage is 20, but allowed 3.  Consider making that variable final if you still need to store its value in advance (before method calls that might have side effects on the original value). [VariableDeclarationUsageDistance]
```

FAILURE: Build failed with an exception.

That's why Amadeus was added in latter lines and not at the beginning of main in the previous example.

Reply:
On my side this code is not checked by the checked (and on Travis it doesn't seem so either). I put it higher so it is easier for users to see it